astropy/coordinates/geodetic.py

```diff
@@ -0,0 +1,136 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import erfa
+
+from astropy import units as u
+from astropy.utils.decorators import format_doc
+
+from .angles import Latitude, Longitude
+from .representation import (
+    BaseRepresentation,
+    CartesianRepresentation,
+)
+
+__all__ = [
+    "BaseGeodeticRepresentation",
+    "WGS84GeodeticRepresentation",
+    "WGS72GeodeticRepresentation",
+    "GRS80GeodeticRepresentation",
+]
+
+ELLIPSOIDS = {}
+"""Available ellipsoids (defined in erfam.h, with numbers exposed in erfa)."""
+# Note: they get filled by the creation of the geodetic classes.
+
+
+def _check_ellipsoid(ellipsoid=None, default="WGS84"):
```

Comment:
This is not used in this module, correct? If so, I think I'd just keep it in `earth.py`